other/phpunit-shim.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+class PHPUnit_Framework_TestCase extends \PHPUnit\Framework\TestCase
```

Comment:
this is not needed if you use phpunit 4.8.36

it contains forward compatible `\PHPUnit\Framework\TestCase` class

```
    "phpunit/phpunit": "^4.8.36|5.*|6.*|7.*|8.*|9.*",
```

Reply:
The converse is the problem: Newer PHPUnit doesn't have the non-namespaced version.

Reply:
Keep in mind: We need tests to run on PHP 5.2.